src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/cosc/CheckAndMarkDefendantPaidInFullCallbackHandler.java

```diff
@@ -64,6 +67,7 @@ private CallbackResponse checkAndMarkDefendantPaidInFull(CallbackParams callback
 
         if (nonNull(judgementPaidDate)) {
             runtimeService.setVariable(caseData.getBusinessProcess().getProcessInstanceId(), SEND_DETAILS_CJES, false);
+            caseData.setJoDefendantMarkedPaidInFullIssueDate(LocalDateTime.now());
```

Comment:
I think it doesnt make sense here. If its already marked paid in full by claimant then it will enter in this if block